manual_jb/content/analyses/housing_household_recovery.md

```diff
@@ -32,17 +32,20 @@ key name | type | name | description
 
 **Input datasets**
```

Comment:
It looks like the parameter "num_cpu" is missing from the input parameters list for housing recovery sequential, can you add this?

Reply:
Added num_cpu!